### PR TITLE
Landing Page Redesign

### PR DESCRIPTION
**Pull request by mike-ferguson**
This PR improves the existing UI, but just the landing page. 

Still to do: 

- [x] Merge with refactor  (#179)
- [x] Navbar burger JS functionality 
- [x]  Make resizing consistent on middle sizes (small are good, middle are a bit odd)
- [x] Update Links to point to right places, especially on footer
- [x] Finalize Text (on landing, about, leaderboard, etc)
Screenshot 2023-06-13 at 1 09 36 PM

### CODE REVIEW

File: static/benchmarks/css/landing_page.sass [hunk 1]
```diff
@@ -0,0 +1,243 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans')
+$new_green: #45C676
+$new_blue: #47B7DE
```
Comment (mschrimpf) [inline static/benchmarks/css/landing_page.sass:3]: let's define these centrally as `brainscore_green` and `brainscore_blue`

File: static/benchmarks/css/landing_page.sass [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  border-radius: 16px
+  border-width: 0
+  font-family: 'Open Sans', sans-serif
+  font-size: 15px
+  font-weight: 600
+  padding: 8px 30px
+  margin-right: 20px
+  background-image: linear-gradient(0deg, $new_blue, $new_green)
+
+.big_green
```
Comment (mschrimpf) [inline static/benchmarks/css/landing_page.sass:37]: how can a color be big?

File: static/benchmarks/css/landing_page.sass [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  margin-top: 100px
+  border-radius: 40px
+
+.image.head_art
+  margin-top: -12px
+  margin-bottom: -12px
+
+.image.logo_green
+  width: 110%
+  max-width: 120%
```
Comment (mschrimpf) [inline static/benchmarks/css/landing_page.sass:73]: make sure this doesn't lead to issues on different devices like phones and tablets

File: benchmarks/templates/benchmarks/landing_page.html [hunk 4]
```diff
@@ -1,11 +1,165 @@
...
-    <meta charset="UTF-8">
-    <title>Placeholder. Actual Page Coming Soon</title>
-</head>
-<body>
-
-<h1>Placeholder. Actual Page Coming Soon</h1>
-</body>
-</html>
\ No newline at end of file
+{% load static %}
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:1]: So far all pages extended the `base.html` page so that global site tags etc are loaded in a unified manner. This is currently lost on the landing page. I think you should have the `landing_page.html` extend the `base.html`, you'll save a lot of redundancy. Note that you can still change the navbar -- using the same code you have now -- because the `block` replaces the corresponding block in `base.html`

File: benchmarks/templates/benchmarks/landing_page.html [hunk 5]
```diff
@@ -1,11 +1,165 @@
...
+{% load compress %}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
+<link rel="stylesheet" href="../static/benchmarks/css/landing_page.css">
+<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,300,800,700,400italic" />
+
+{# own sources #}
+{% compress js %}
+    <script defer src="{% static "/benchmarks/js/navbar.js" %}" type="text/javascript" charset="utf-8"></script>
+{% endcompress %}
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:10]: this can go

File: benchmarks/templates/benchmarks/landing_page.html [hunk 6]
```diff
@@ -1,11 +1,165 @@
...
+                        <a class="social_icon navbar-item" href="https://www.twitter.com/brain_score">
+                            <img class="navbar_element image" id="twitter_icon" src="/static/benchmarks/img/new_ui_elements/twitter_icon.png"/>
+                        </a>
+                    </div>
+                </div>
+        </div>
+    </div>
+</nav>
+{% endblock %}
+
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:41]: following suggestion, this should be `{% block content %}` and you then don't have to start with root-level divs

File: benchmarks/templates/benchmarks/landing_page.html [hunk 7]
```diff
@@ -1,11 +1,165 @@
...
+<div class="landing container">
+    <div class="columns">
+        <div class="left_side column is-half has-text-centered-mobile">
+            <img class="logo_green image is-inline-block"  src="/static/benchmarks/img/new_ui_elements/logo_name_full.png" />
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:45]:
```suggestion
            <img class="logo_green image is-inline-block"  src="{% static "/benchmarks/img/new_ui_elements/logo_name_full.png" %}" />
```
Use django to infer the static path, do not type yourself

Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:45]: the folder `new_ui_elements` needs renaming. Why not just place the new elements inside `img`? Or `img/landing_page/`?

File: benchmarks/templates/benchmarks/landing_page.html [hunk 8]
```diff
@@ -1,11 +1,165 @@
...
+            <h3 class="landing_heading is-size-6 is-size-4-widescreen is-size-4-mobile">Integrative Benchmarks | Models at Scale </h3>
+            <p class="text-default-small is-size-7 is-size-6-widescreen has-text-justified-mobile">The Brain-Score platform aims to yield strong computational models of the
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:47]:
```suggestion
            <p class="text-default-small is-size-7 is-size-6-widescreen has-text-justified-mobile">
                The Brain-Score platform aims to yield strong computational models of the
```

File: benchmarks/templates/benchmarks/landing_page.html [hunk 9]
```diff
@@ -1,11 +1,165 @@
...
+                brain and mind. We enable researchers to quickly get a sense of the alignment of their model(s) to
+                currently dozens of neural and behavioral measurements, and provide these
+                models to experimentalists to prototype new experiments and make sense of biological data. </p>
+            <div class="domain_buttons container">
+                <button onclick="location.href='../vision'" type="button" class="button domain_button is-size-7 is-size-6-widescreen" >Vision</button>
+                <button onclick="location.href='../language'" type="button" class="button domain_button is-size-7 is-size-6-widescreen" >Language</button>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:53]: The `../` in front of `vision` and `language` concerns me, why are those necessary?

File: benchmarks/templates/benchmarks/landing_page.html [hunk 10]
```diff
@@ -1,11 +1,165 @@
...
+            </div>
+        </div>
+        <div class="column is-half">
+            <img class="head_art image"  src="/static/benchmarks/img/new_ui_elements/head_art_1.png" />
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:57]: use django for static

File: benchmarks/templates/benchmarks/landing_page.html [hunk 11]
```diff
@@ -1,11 +1,165 @@
...
+        </div>
+    </div>
+</div>
+
+<div class="main_body container">
+    <div class="about container">
+        <div class="columns">
+            <div class="column">
+                <img class="collab_graphic image"  src="/static/benchmarks/img/new_ui_elements/collaboration.png" />
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:66]: use django for static

File: benchmarks/templates/benchmarks/landing_page.html [hunk 12]
```diff
@@ -1,11 +1,165 @@
...
+                     <p class="benefits_info is-size-7 is-size-6-widescreen">Download and Experiment. Completely free.</p>
+                    <button onclick="location.href='https://www.github.com/brain-score'" type="button" class="button benefit_button is-size-7 is-size-6-widescreen " >Github</button>
+                 </div>
+             </div>
+             <div class="column is-one-third">
+                 <div class="benefits_card container">
+                    <img class="benefit_icon image"  src="/static/benchmarks/img/new_ui_elements/access_icon.png" />
+                    <h3 class="benefits_heading is-size-5 is-size-3-widescreen">Accessible</h3>
+                    <p class="benefits_info is-size-7 is-size-6-widescreen">Interact seamlessly with our curated leaderboard.</p>
+                    <button onclick="location.href='../profile'" type="button" class="button benefit_button is-size-7 is-size-6-widescreen" >Submit</button>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:103]: `../`

File: benchmarks/templates/benchmarks/landing_page.html [hunk 13]
```diff
@@ -1,11 +1,165 @@
...
+                    <button onclick="location.href='https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html#'" type="button" class="button tutorial_button" >Tutorials</button>
+                    <button onclick="location.href='../profile/'" type="button" class="button hollow_button" >Submit</button>
+                 </div>
+            </div>
+        </div>
+    </div>
+    <div class="main_footer container">
+        <div class="columns has-text-centered-mobile">
+            <div class="column has-text-centered ">
+                <img class="white_logo image is-inline-block"  src="/static/benchmarks/img/new_ui_elements/White_logo_full.png" />
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:136]: use django for static

File: benchmarks/templates/benchmarks/landing_page.html [hunk 14]
```diff
@@ -1,11 +1,165 @@
...
+            </div>
+            <div class="footer_middle column has-text-centered">
+                <div class="column">
+                    <button onclick="location.href='https://github.com/brain-score/brain-score/issues'" type="button" class="button simple_button" >Support</button>
+                    <button onclick="location.href='https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html#frequently-asked-questions'" type="button" class="button simple_button" >FAQ</button>
+                </div>
+                <div class="column">
+                    <button onclick="location.href='https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html#'" type="button" class="button simple_button" >Tutorials</button>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:144]: what about the benchmark tutorial? I'd rather we link to some central place

File: benchmarks/templates/benchmarks/landing_page.html [hunk 15]
```diff
@@ -1,11 +1,165 @@
...
+    </div>
+    <div class="main_footer container">
+        <div class="columns has-text-centered-mobile">
+            <div class="column has-text-centered ">
+                <img class="white_logo image is-inline-block"  src="/static/benchmarks/img/new_ui_elements/White_logo_full.png" />
+            </div>
+            <div class="footer_middle column has-text-centered">
+                <div class="column">
+                    <button onclick="location.href='https://github.com/brain-score/brain-score/issues'" type="button" class="button simple_button" >Support</button>
+                    <button onclick="location.href='https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html#frequently-asked-questions'" type="button" class="button simple_button" >FAQ</button>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:141]: what about benchmark or general FAQs?

File: benchmarks/templates/benchmarks/landing_page.html [hunk 16]
```diff
@@ -1,11 +1,165 @@
...
+                </div>
+                <div class="column">
+                    <button onclick="location.href='https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html#'" type="button" class="button simple_button" >Tutorials</button>
+                    <button onclick="location.href='../profile/'" type="button" class="button simple_button" >Log In</button>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:145]: `../`

File: benchmarks/templates/benchmarks/landing_page.html [hunk 17]
```diff
@@ -1,11 +1,165 @@
...
+                </div>
+                 <div class="column">
+                    <button onclick="location.href='https://github.com/brain-score/brain-score#references'" type="button" class="button simple_button" >Citations</button>
+                    <button onclick="location.href='https://github.com/brain-score/brain-score#license'" type="button" class="button simple_button" >License</button>
+                </div>
+                <div class="column">
+                    <button onclick="location.href='https://quest.mit.edu'" type="button" class="button simple_button" >Sponsors</button>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:152]:
```suggestion
```
not for now

File: benchmarks/templates/benchmarks/landing_page.html [hunk 18]
```diff
@@ -1,11 +1,165 @@
...
+                </div>
+            </div>
+            <div class="column">
+                <h1 class=join_text>Join our Newsletter!</h1>
+                <div class="control">
+                    <input class="input" type="text" placeholder="Sign up Coming Soon :)" disabled>
+                    <button onclick="location.href='../profile'" type="button" class="button white_button"
+                            title="Disabled button" disabled >Sign Up</button>
+                </div>
+            </div>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/landing_page.html:163]:
```suggestion
```
we don't have a newsletter

File: benchmarks/views/user.py [hunk 19]
```diff
@@ -135,6 +135,12 @@ def get(self, request):
         return HttpResponseRedirect('../')
 
 
+class Landing(View):
+
```
Comment (mschrimpf) [inline benchmarks/views/user.py:139]:
```suggestion
```

File: static/benchmarks/css/landing_page.css [hunk 20]
```diff
@@ -0,0 +1,303 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans");
```
Comment (mschrimpf) [inline static/benchmarks/css/landing_page.css:1]: This file is auto-generated, no? Do not commit.

File: static/benchmarks/css/landing_page.css.map [hunk 21]
```diff
@@ -0,0 +1 @@
...AAQ;AAMR;EACE;;;AAEF;EACE;EACA;;;AAEF;EACE;EACA;;;AAEF;EACE;;;AAIF;EACE;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE,OAxCiB;EAyCjB;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;;;AAEF;EACE;EACA;;;AAEF;EACE;EACA;EACA;;;AAEF;EACE;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE;;;AAEF;EACE;EACA;;;AAIF;EACE;EACA;EACA;;;AAIF;EACE;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;;;AAEF;EACE,OA/HiB;EAgIjB;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;;;AAIF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA,OApLiB;EAqLjB;;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;;;AAIF;EACE;EACA;;;AAEF;EACE;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA;EACA;;;AAEF;EACE;EACA;EACA;EACA,OAtNiB;;;AA0NnB;EACE;IACE;;;EACF;IACE;IACA;IACA;;;EACF;IACE;IACA;;;EACF;IACE;;;AAGJ;EACE;IACE;IACA;IACA;;;AAGJ;EACE;IACE;IACA;;;AAGJ;EACE;IACE;IACA;;;EACF;IACE;IACA;IACA;;;EACF;IACE;IACA;;;EACF;IACE;IACA;IACA;;;EACF;IACE;;;AAIJ;EACE;IACE","file":"landing_page.css"}
```
Comment (mschrimpf) [inline static/benchmarks/css/landing_page.css.map:1]: auto-generated, do not commit file

File: static/benchmarks/css/landing_page.sass [hunk 22]
```diff
@@ -0,0 +1,271 @@
...
+  height: 35px
+
+.image.b-w_logo
+  width: 50px
+  height: auto
+  padding-bottom: 30px
+  margin: auto
+
+
+// leaderboard page (on landing)
```
Comment (mschrimpf) [inline static/benchmarks/css/landing_page.sass:163]:
```suggestion
// leaderboard section (on landing)
```

**Comment by mike-ferguson** [comment]
Login Page (preliminary): 
Screenshot 2023-08-09 at 3 09 06 PM

**Comment by mike-ferguson** [comment]
Screenshot 2023-11-21 at 9 34 21 AM
Preliminary Tutorial page (`/tutorial`)

**Review by mschrimpf — APPROVED** [review]
